pyChilizer.tab/Template.panel/template2.stack/List Walls.pushbutton/script.py

```diff
@@ -108,19 +139,16 @@ def change_wall_type(wall, wall_type):
 view = revit.active_view
 level = get_level_from_view(view)
 
-line_length = 6
```

Comment:
moved this to the top as CONSTANTS grabbed from the config file

Reply:
Hey man! Thanks for the PR! 

Let me review and will merge.

Reply:
Options do come up, but appear blank @jmcouffin . I have just tested it with the Simple Revit Sample project. Maybe something I am not getting in terms of how it is to be used?

<img width="1140" alt="image" src="https://user-images.githubusercontent.com/5354594/227740513-97bd8a78-77f8-4116-ad7c-fa396d8ab44e.png">